save.go

```diff
@@ -354,14 +354,35 @@ func copyFile(dst, src string) error {
 }
 
 func copyWithoutImportComment(w io.Writer, r io.Reader) error {
-	sc := bufio.NewScanner(r)
-	for sc.Scan() {
-		_, err := w.Write(append(stripImportComment(sc.Bytes()), '\n'))
+	var eof bool
+	b := bufio.NewReader(r)
+	for {
+		l, err := b.ReadBytes('\n')
 		if err != nil {
-			return err
+			if err == io.EOF {
```

Comment:
It would be pretty cool to pull this out in front of the surrounding if stmt:

```go
l, err := b.ReadBytes('\n')
eof := err == io.EOF
if err != nil && err != io.EOF {
  return err
}
```